R/add_phylopic_tree.R

```diff
@@ -0,0 +1,84 @@
+#' Add PhyloPics to a plotted phylogenetic tree
+#'
+#' Specify existing images, taxonomic names, or PhyloPic uuids to add PhyloPic
+#' silhouettes alongside the associated leaves of a phylogenetic tree.
+#'
+#' @inheritParams add_phylopic
+#' @param tree The phylogenetic tree object of class `phylo` on which to add
+#' the silhouette.
+#' @param tip The tip labels against which to add the silhouettes.
+#' @param relWidth The width of each silhouette relative to the plotting area.
```

Comment:
It's not clear to me why there is this argument when there is already the `width` argument?

Reply:
The coordinate space used to construct a tree is not visible or obvious to the user; it is a function of the number of tips (and edge length?).  `relWidth` gives a consistent way to specify the size of a silhouette across plots – even if the number of tips in a plot is not known in advance.  For instance, my workflow plots a consensus tree, then a second consensus after removing rogue taxa; the number of rogue taxa that will be found is not known in advance.  `relWidth` allows me to have a consistent silhouette size between plots without having to infer the plotting coordinates myself for each plot.

Reply:
OK, I'm sold, but then to be consistent, I think it would be good to have both padding and relPadding as you suggest below.